release.py

```diff
@@ -0,0 +1,264 @@
+                    )
+                elif filename == "__init__.py":
+                    updated_line = re.sub(
+                        "__version__ ?=.*# pragma: no cover",
+                        "__version__ = '{version}'  # pragma: no cover".format(version=new_version),
+                        line,
+                    )
+                elif filename == "setup.py":
+                    updated_line = re.sub(
+                        "version=.*",
+                        "version='{version},'".format(version=new_version),
+                        line,
+                    )
+                else:
+                    # Shouldn't get here
+                    raise OldVersionException("Unexpected file {}".format(filename))
+                file_lines.append(updated_line)
+            else:
+                file_lines.append(line)
+
+    if updated:
+        # Replace contents of file with updated version
+        with open(version_filepath, "w") as f:
+            for line in file_lines:
+                f.write(line)
+        return old_version
+
+    # Unable to find old version for this file, but maybe there's another one
+
+
+def update_version(new_version):
+    """Update the version from the project and return the old one, or raise an exception if none is found"""
+    print("Updating version...")
+    version_files = ('settings.py', '__init__.py', 'setup.py')
+    for version_filename in version_files:
+        for root, _, filenames in os.walk("."):
+            if version_filename in filenames:
+                result = update_version_in_file(root, version_filename, new_version)
+                if result:
+                    return result
+
+    raise OldVersionException("Unable to find previous version number")
+
+
+def create_release_notes(old_version, with_checkboxes):
+    """Returns the release note text for the commits made for this version"""
+    if with_checkboxes:
+        filename = "release_notes.ejs"
+    else:
+        filename = "release_notes_rst.ejs"
+
+    return check_output([
+        "git-release-notes",
+        "v{}..master".format(old_version),
+        os.path.join(SCRIPT_DIR, "util", filename),
+    ]).decode()
+
+
+def verify_new_commits(old_version):
+    """Check if there are new commits to release"""
+    if int(check_output(["git", "rev-list", "--count", "v{}..master".format(old_version)])) == 0:
+        raise Exception("No new commits to put in release")
+
+
+def update_release_notes(old_version, new_version):
+    """Updates RELEASE.rst and commits it"""
+    print("Updating release notes...")
+
+    release_notes = create_release_notes(old_version, with_checkboxes=False)
+
+    release_filename = "RELEASE.rst"
+    try:
+        with open(release_filename) as f:
+            existing_note_lines = [line for line in f.readlines()]
+    except FileNotFoundError:
+        existing_note_lines = []
+
+    with open(release_filename, "w") as f:
+        f.write("Release notes\n")
```

Comment:
Could you capitalize "notes" so it matches the previous template?

Reply:
Yes